### DISCUSSION TITLE
视频控制面板将实现更多的操作方式

### DISCUSSION THREAD

**Discussion by ProJend** [ywmoyue/biliuwp-lite#902] [General]
重写后的视频控制面板将会实现更多的操作方式，触摸和鼠标操作互不影响，且不再提供复杂的相关设置。
大家有想实现的功能都可以提出来，我会择优完成

**Comment by ywmoyue** [comment]
关联PR https://github.com/ywmoyue/biliuwp-lite/pull/898

**Comment by ProJend** [comment]
> 关联PR https://github.com/ywmoyue/biliuwp-lite/pull/898

我打算开个新PR

**Comment by ywmoyue** [comment]
这个事情可能和我原本计划的播放器重构冲突，不过没关系，短时间我计划的播放器重构也没时间做
